vars/parseStageInfo.groovy

```diff
@@ -183,7 +183,7 @@ def call(Map config = [:]) {
                 if (env.BRANCH_NAME.startsWith("weekly-testing")) {
                   tag = "full_regression"
                 } else {
-                  tag = "daily_regression"
+                  tag = "pr daily_regression"
```

Comment:
This will work.  Although I think it's more transparent to add the `daily_regression` tag to the tests that we want to include in the daily testing (i.e. all of the `pr` tagged tests currently) run.  That would not leave somebody who doesn't have the _tribal knowledge_ that daily testing includes the `pr` tag not understanding why a test that does not have the `daily_regression` tag (which does exist, adding to the confusion) is in the daily testing run.

It also allows for future flexibility in removing a particular `pr` tagged test from the daily run.

While I understand my suggestion here has been made and rejected, I just wanted to note it for the record.